gensim/models/word2vec.py

```diff
@@ -432,6 +432,8 @@ def __init__(
 
         """
 
+        self.load = call_on_class_only
```

Comment:
This will leave the variable unbound, leading to potential confusion / serialization issues.

Proper way would be to bind it properly with `types.MethodType` or somesuch.